epub33/rs/index.html

```diff
@@ -1072,14 +1072,14 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 							regular and center-spread pages.</p>
 					</div>
 
-					<p id="page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
+					<p id="page-layout-both" data-tests="#page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
 							<a href="#def-layout-pre-paginated">pre-paginated</a> one, the reflowable one SHOULD start
 						on the next page &#8212; as defined by the <a
 							href="https://www.w3.org/TR/epub-33/#attrdef-spine-page-progression-direction"
 								><code>page-progression-direction</code></a> [[EPUB-33]] &#8212; when it lacks a
 							<code>rendition:page-spread-*</code> property value.</p>
 
-					<p id="page-spread-flow">If the reflowable spine item has a <code>rendition:page-spread-*</code>
+					<p id="page-spread-flow" data-tests="#page-layout-both-flow">If the reflowable spine item has a <code>rendition:page-spread-*</code>
```

Comment:
Same

Reply:
#fxl-page-spread-flow? Kinda similar to -reflowable above, but I think this ID is not so bad, and it's all arbitrary anyway.